design-library/src/components/BccTabs/BccTabs.css

```diff
@@ -0,0 +1,62 @@
+@layer components {
+    .bcc-tabs {}
+
+    .bcc-tabs-bar {
+        @apply relative flex items-end gap-4 md:gap-8;
+    }
+
+    .bcc-tabs-bar::before {
+        @apply absolute bottom-0 inset-x-0 border-b-2 border-on-primary;
+        content: '';
+    }
+
+    @media (max-width: theme('screens.sm')) {        
+        .bcc-tabs-bar--2 .bcc-tabs-bar-item {
+            @apply w-1/2 md:w-min;
+        }
+
+        .bcc-tabs-bar--3 .bcc-tabs-bar-item {
+            @apply w-1/3;
+        }
+
+        .bcc-tabs-bar--4 .bcc-tabs-bar-item {
+            @apply w-1/4;
+        }
+    }
```

Comment:
This seems pretty specific: what happens for tabs with >= 4 items?

Reply:
They should just scroll. But now that I see this again, I realise it is wrong because it is supposed to be min-width. :D 
The reason is purely that if there are only 2-4 items and if possible fit them together on the mobile screen. Four items is probably just an icon.